README.md

```diff
@@ -777,37 +777,42 @@ for signing. The bot calls the script with the two arguments:
 The section `[architecturetargets]` defines for which targets (OS/SUBDIR), (for example `linux/x86_64/amd/zen2`) the EESSI bot should submit jobs, and which additional `sbatch` parameters will be used for requesting a compute node with the CPU microarchitecture needed to build the software stack.
 
 ```ini
-arch_target_map = {
```

Comment:
Would it be useful to include an example of how to migrate from a semi-realistic `arch_target_map` to `node_type_map`?

Reply:
clarified in https://github.com/EESSI/eessi-bot-software-layer/commit/be7efe42036eaf4bae399bfe1b789e65a19ccbc5